src/components/TemplateLibrary/index.js

```diff
@@ -28,7 +35,7 @@ const LibraryComponent = () => {
       items = {templates}
       // TODO
       onPrimaryButtonClick={() => console.log('Action to add this template to contract')}
-      onSecondaryButtonClick={() => console.log('Action to view the details')}
+      onSecondaryButtonClick={template => goToTemplateDetail(template)}
```

Comment:
This could be reduced to `onSecondaryButtonClick={goToTemplateDetail}`

Reply:
I just did this to keep both attributes (`onPrimaryButtonClick` and `onSecondaryButtonClick`) symmetrical :sweat_smile: